src/views/Operations/Firmware/Firmware.vue

```diff
@@ -22,9 +22,19 @@
     </b-row>
 
     <!-- Update firmware-->
-    <page-section
-      :section-title="$t('pageFirmware.sectionTitleUpdateFirmware')"
-    >
+    <page-section :section-title="$t('pageFirmware.sectionTitleUpdateFirmware')"
+      ><b-row>
+        <b-col sm="14" md="10" xl="6">
+          <alert variant="info" class="mb-5">
```

Comment:
Lets just show this message when we have a value being returned in the redfish response.